gluten-substrait/src/main/scala/org/apache/spark/sql/execution/datasources/GlutenWriterColumnarRules.scala

```diff
@@ -39,10 +39,6 @@ object GlutenWriterColumnarRules {
     "org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat" -> "parquet"
   )
   private def getNativeFormat(cmd: DataWritingCommand): Option[String] = {
-    if (!BackendsApiManager.getSettings.enableNativeWriteFiles()) {
```

Comment:
From a code readability perspective, it makes more sense to keep this check within the getNativeFormat method.

Reply:
Because `getNativeFormat` is a private method, it is safe.